test/controllers/JourneySetupControllerSpec.scala

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
Can you add some tests to cover the result being  BAD requests as returned in the code please.